packages/regal/.fvm/fvm_config.json

```diff
@@ -1,4 +1,3 @@
 {
-  "flutterSdkVersion": "3.10.6",
-  "flavors": {}
+  "flutterSdkVersion": "stable"
```

Comment:
stable will mean different depending on dev's laptop config

Reply:
i'll keep it to 3.13.9 as minimum then

Reply:
better yet i'll remove fvm config from git history @vamsi-kleargroup bro?

Reply:
nah..its better to have..everyone can update if we updated it